client/bookmarkUtil.js

```diff
@@ -46,11 +46,15 @@ const getPrevOrderFromNextOrder = (lastNumber, order) => {
  * @param {String} nextOrder
  * @returns {String}
  */
-module.exports.getBookmarkOrder = (prevOrder, nextOrder) => {
+module.exports.getBookmarkOrder = (prevOrder, nextOrder, parentOrder) => {
   let prevOrderSplit = prevOrder.split('.')
   let nextOrderSplit = nextOrder.split('.')
   if (prevOrderSplit.length === 1 && nextOrderSplit.length === 1) {
-    throw new Error(`Invalid previous and next orders: ${prevOrderSplit} and ${nextOrderSplit}`)
+    if (!parentOrder) {
```

Comment:
we should handle the case where all 3 are blank (first node added) - we really have no use or need for the base order in the client and I think we should encapsulate all the logic here so the client doesn't have to know or care about how the order works, so I think we should store it in the lib and use it for this case

Reply:
we have a workaround for now in brave-core so that could be done in a follow up